core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+package org.apache.spark.status.protobuf;
+
+message JavaDate {
+  // The number of milliseconds since January 1, 1970, 00:00:00 GMT
+  optional int64 date = 1;
+}
+
+enum JobExecutionStatus {
+  RUNNING = 0;
```

Comment:
do we nee to add `UNSPECIFIED` to follow protobuf style guideline? cc @amaliujia

Reply:
probably it's fine as these proto message are only used by Spark internally.

Reply:
I didn't get it. There is UNKNOWN status already. There is always a value set for JobExecutionStatus.

Reply:
FYI that is a from the official style guide that 0 is used for `UNSPECIFIED` and keep `UNSPECIFIED`: https://developers.google.com/protocol-buffers/docs/style#enums

Reply:
+1, please reserve `0` for unspecified to allow for evolution.

Reply:
Sure, updated